R/IDateTime.R

```diff
@@ -240,6 +240,30 @@ rep.ITime = function (x, ...)
   class(y) = "ITime"   # unlass and rep could feasibly not copy, hence use class<- not setattr()
   y
 }
+                           
+round.ITime <- function(x, digits = c("hours", "minutes")) 
```

Comment:
`round.POSIXt` is using `units` argument, which is probably more intuitive than `digits`